config/clients/go/template/README_retries.mustache

```diff
@@ -0,0 +1,26 @@
+By default API requests are retried up to 15 times on 429 and 5xx errors with a minimum wait time of 100 milliseconds and credential requests are retried up to 3 times on 429 and 5xx errors with a minimum wait time of 50 milliseconds.
```

Comment:
In all of these, instead of hardcoding 15, can you use the value from config?

Reply:
Ahh didn't realise that was a thing! I'll also extract the client credentials into a configuration setting